### DISCUSSION TITLE
The development environment selects the node in the browser, and the execution animation is very stuck

### DISCUSSION THREAD

**Discussion by hsiangleev** [tailwindlabs/tailwindcss#3292] [Help]
1. In the development environment, when the page node is selected in the browser and the CSS animation is executed, the page will be stuck and the cpu will skyrocket.
2. When the "--tw-shadow: 0 0 #0000;" attribute is cancelled, the animation is executed and the page is stuck for about 10 seconds
3. The packaged code did not find this problem

**Comment by simonswiss** [comment]
Hey! Are you able to share some code to look at?
